### DISCUSSION TITLE
New RPI 4 V1.5 serialbattery does not install

### DISCUSSION THREAD

**Discussion by Attekap** [Louisvdw/dbus-serialbattery#385] [Q&A] [answered]
after running sh installrelease.sh i don't get a error or conformation jut a new row where I can write new commands. 

I both Offgrid garage and Louis videos they get a output after the sh installrelease.sh command similar to 
cp: omitting directory /data/etc/......
I don't get anything. 
when I go to Troubleshoot the pi shows up at VRM but does not show anything when typing in tail /data/log/serial-starter/current or /data/log/dbus-serialbattery.ttyUSB0/

tried with Venus 2.92 and 3.00-14 also tried with dbus-serialbattery V0.14 V0.13 and V0.12 but get the same result.
I also upgraded the pi eeprom to latest version and tried again but still the same. 
Looks like the install does not complete.

Anyone that have similar problem and got it working in the end?

**Comment by Louisvdw** [comment, marked as answer]
The omitting warning was fixed (hidden) in the newer versions, so it is not a bad thing that it show no errors for you. 

The log files will only populate if you do plug in a serial device. Have you connected your BMS to one of the Pi's USB ports?

**Reply by Attekap** [reply]
I think that the serial device is broken? it does not show up when i plug it in on my laptop (without being connected to the bms) when i run the commands  lsusb and usb-devices. It should show up right? i order 2 new ones and 2 different models.

**Reply by Attekap** [reply]
New serial device arrived today and everything works.
